packages/nhost_dart/pubspec.yaml

```diff
@@ -23,7 +23,7 @@ dependencies:
   nhost_graphql_adapter: ^4.0.5
 dev_dependencies:
   fake_async: ^1.3.1
-  graphql: ^5.1.3
+  graphql: ^5.2.0-beta.9
```

Comment:
Is the version upgrade to `graphql: ^5.2.0-beta.9` required for the this example ?

Reply:
No. I messed up the merging of main branch